UnitTests/expected/Storyboards-osx-Message-Defaults.swift.out

```diff
@@ -46,22 +46,22 @@ struct StoryboardScene {
 
     case MessageDetailsScene = "MessageDetails"
     static func instantiateMessageDetails() -> NSViewController {
-      return StoryboardScene.Message_Osx.MessageDetailsScene.controller()
+      return StoryboardScene.Message_Osx.MessageDetailsScene.controller() as! NSViewController
```

Comment:
After a rebase on master, here the output: `Done linting! Found 0 violations, 0 serious in 1 file.`
I'm using zsh, but I got no warnings, I have them only when building when Xcode. Did you try to build using `feature/storyboards_osx`branch? I don't think that will change anything but...